ext/opcache/ZendAccelerator.c

```diff
@@ -899,7 +895,11 @@ static inline void kill_all_lockers(struct flock *mem_usage_check)
 
 static inline int accel_is_inactive(void)
 {
-#ifdef LOCKVAL
+#ifdef ZEND_WIN32
+	/* TODO: the Windows-specific code path is broken because
```

Comment:
As per comment https://github.com/php/php-src/pull/10276#issuecomment-1384235011 this doesn't seem to be accurate.

Reply:
Yes, that explains it, thanks @cmb69 - I'll adjust the comment.